plugins/modules/terraform.py

```diff
@@ -165,6 +165,12 @@
       - Restrict concurrent operations when Terraform applies the plan.
     type: int
     version_added: '3.8.0'
+  no_color:
+    description:
+      - If true, suppress color codes in output from Terraform commands.
+      - If false, allows Terraform to use color codes in its output.
+    type: bool
+    default: true
```

Comment:
```suggestion
    default: true
    version_added: 11.0.0
```

Reply:
@russoz 10.7.0 is already out since yesterday, so it should be 11.0.0 :)

Reply:
oopsie